### ISSUE TITLE
Maximum/Minimum error messages seem to be reversed

### ISSUE THREAD

**Issue by glamic-ep** [swaggest/php-json-schema#141]
In [`Swaggest\JsonSchema\Schema::processNumeric`](https://github.com/swaggest/php-json-schema/blob/1d7cce2dac6844a8a612672ad5936e8c4145e96a/src/Schema.php#L355) the error messages regarding maximum/minimum and exclusive maximum/minimum seem to be reversed. At the maximum it says "Value less than ..." and at the exclusive maximum it says "Value less or equal than...". I think it needs to be the other way round. The same for minimum.

**Comment by kemal-xx** [comment]
Yes. I just found out about this today. The messages are wrongly reversed between the two.
